networking/topology.md

```diff
@@ -30,15 +30,15 @@ In the future, this VNet might hold more subnets like [ACI Provider instance] su
+| AKS System and User Nodepool Subnet         | 2            | 5                  | 200                  | 10         | [30]                     | 100           | 0                   | 30                    | 60                        | 5                                        | 0                   | 372                   | 982                | 22                 | 10.240.0.0/22  | 10.240.0.0  | 10.240.3.255  |
+| AKS Internal Load Balancer Services Subnet  | -            | -                  | -                    | -          | 5                        | 100           | 100                 | 0                     | 5                         | 5                                        | 0                   | 10                    | 10                 | 28                 | 10.240.4.0/28  | 10.240.4.0  | 10.240.4.15   |
+| Private Link Endpoint Subnet                | -            | -                  | -                    | -          | -                        | 100           | 100                 | 0                     | 0                         | 5                                        | 2                   | 7                     | 7                  | 28                 | 10.240.4.32/28 | 10.240.4.32 | 10.240.4.47   |
+| Azure Application Gateway Subnet            | -            | [251]              | -                    | -          | -                        | 100           | 100                 | 0                     | 0                         | 5                                        | 0                   | 256                   | 256                | 24                 | 10.240.5.0/24  | 10.240.5.0  | 10.240.5.255  |
+| Gateway Subnet (GatewaySubnet)              | -            | [27<sup>1</sup>]   | -                    | -          | -                        | 100           | 100                 | 0                     | 0                         | 5                                        | 0                   | 32                    | 32                 | 27                 | 10.200.0.64/27 | 10.200.0.64 | 10.200.0.95   |
+| Azure Firewall Subnet (AzureFirewallSubnet) | -            | [59]               | -                    | -          | -                        | 100           | 100                 | 0                     | 0                         | 5                                        | 0                   | 64                    | 64                 | 26                 | 10.200.0.0/26  | 10.200.0.0  | 10.200.0.63   |
+| Azure Bastion Subnet (AzureBastionSubnet)   | -            | [50<sup>2</sup>]   | -                    | -          | -                        | 100           | 100                 | 0                     | 0                         | 5                                        | 0                   | 64                    | 64                 | 26                 | 10.200.0.128/26 | 10.200.0.128 | 10.200.0.191  |
```

Comment:
The superscript here seems to indicate that there is more information below, but # 2 in Additional Considerations is talking about a different subnet.  Not new for this PR, but maybe we can clean that up?

Reply:
The superscript link works, it's literally the whole text, when clicked, goes to that data source, but I see what you mean how it could be confused with the numbered list below.  Let me fix that!  Good idea.

Reply:
Double fix, removed the (now) unnecessary superscript -- they were used as differentiators before, and also converted the numbered list into a bullet list.